projects/Amlogic/initramfs/platform_init

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
I don't think this file is needed

Reply:
I'm not an expert on this but `platform_init` is run, not sourced so perhaps this should be kept? Also it follows other projects: https://github.com/LibreELEC/LibreELEC.tv/blob/master/projects/RPi/initramfs/platform_init#L1

Reply:
Sorry, I misread that you meant line.

It might be placebo but GUI animations are smoother with these tunables set, they keep CPU at higher freq a bit longer so it doesn't ramp down when animation is in progress.

Reply:
this shouldn't be project specific then. We should make it generic for all platforms using udev rules

Reply:
Done: https://github.com/LibreELEC/LibreELEC.tv/pull/3538